integration/terraform/ec2/README.md

```diff
@@ -236,7 +246,7 @@ upload: ./terraform.tfstate to s3://***/integration-test/local-stack-terraform-s
 
 In this example, you should keep track of `ec2-35-87-254-148.us-west-2.compute.amazonaws.com`
 
-Start the linux integration tests:
+Start the linux integration tests (example):
```

Comment:
IMO, I would see this as a `start the linux integration  test` more than an example because we don't provide any sample inputs. However, I would say as an example would be a good idea and comments for each input.

Reply:
I put "example" here because I moved this outside of the linux folder and don't want to confuse someone who is only concerned with running Windows tests (when those eventually get written)